lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -144,6 +144,36 @@ def route(controller:, action:)
         nil
       end
 
+      # Delegates a notification to a server add-on
+      sig { params(server_addon_name: String, request_name: String, params: T.untyped).void }
+      def delegate_notification(server_addon_name, request_name, **params)
```

Comment:
I feel like kwargs are nicer since it's documenting the arguments. Wdyt?

Reply:
I don't mind. The LSP can now show signature help, so you see the parameters in the editor anyway.

@andyw8 any opinions?